docs/Authenticating-Users-via-LDAP.md

```diff
@@ -73,7 +69,14 @@ Rebuild the WAR file and re-deploy:
 
 ```
 mvn -DskipTests clean install
-cp portal/target/cbioportal.war $CATALINA_HOME/webapps/
+java \
+    -Dauthorization=true \
+    -Dauthenticate=ldap \
+    -Ddbconnector=dbcp \
+    -jar \
+    portal/target/dependency/webapp-runner.jar \
+    --path /cbioportal \
+    portal/target/cbioportal.war
```

Comment:
you can deploy on localhost:8080 instead of localhost:8080/cbioportal when not using `--path`. This might be better as default but I thought this would avoid breaking existing docs

Reply:
We've definitely been considering that as well, but not yet made a pull request for that same reason—nor documented it downstream in [thehyve/cbioportal-docker](https://github.com/thehyve/cbioportal-docker).

Reply:
@fedde-s maybe when i update the docker documentation i can do a find and replace of `localhost:8080/cbioportal` to `localhost:8080` (figured to do that next).

Reply:
i updated the path in this PR